cmd/dashboard/main.go

```diff
@@ -38,6 +38,15 @@ import (
 
 const csrfTokenLength = 32
 
+var (
+	help             = flag.Bool("help", false, "Prints defaults")
+	kubeConfigPath   = flag.String("kube-config", "", "Path to kube config file")
+	portNumber       = flag.Int("port", 8080, "Dashboard port number")
+	readOnly         = flag.Bool("read-only", false, "Enable or disable read only mode")
+	webDir           = flag.String("web-dir", "", "Dashboard web resources dir")
+	csrfSecureCookie = flag.Bool("csrf-secure-cookie", true, "Enable or disable CSRF secure cookie")
```

Comment:
'Enable or disable Secure attribute on the CSRF cookie'

The cookie is always present, this just determines whether we set the Secure attribute to prevent the browser sending it on requests over insecure connections.

Reply:
Changed